waku.md

```diff
@@ -322,16 +328,67 @@ Waku is a different subprotocol from Whisper so it isn't directly compatible. Ho
 
 It is desirable to have a strategy for maintaining forward compatibility between `waku/0` and future version of waku. Here we outline some concerns and strategy for this.
 
-
-## Security considerations
+## Appendix A: Security considerations
 
 There are several security considerations to take into account when running Waku. Chief among them are: scalability, DDoS-resistance and privacy. These also vary depending on what capabilities are used, such as mailserver, light node, and so on.
 
-### Light node privacy
+### Scalability and UX
+
+**Bandwidth usage:**
+
+In version 0 of Waku, bandwidth usage is likely to be an issue. For more investigation into this, see the theoretical scaling model described [here](https://vac.dev/fixing-whisper-with-waku).
```

Comment:
Would it be worth it to include the bandwidth usage doc in this repo somewhere? To me feels like is useful information and can start giving us a framework to analyse bandwidth in the future.

Reply:
How do you think it should be included? We can link to the https://github.com/vacp2p/research/tree/master/whisper_scalability repo if that seems better. Not sure it belongs in specs repo. Agree re iterating on it.

Reply:
def. version control, I would not mind personally having it as another appendix, but as long as it is version controlled I am happy

Reply:
updated link to version controlled


In version 0 of Waku, bandwidth usage is likely to be an issue. For more investigation into this, see the theoretical scaling model described [here](https://github.com/vacp2p/research/tree/dcc71f4779be832d3b5ece9c4e11f1f7ec24aac2/whisper_scalability).